tools/FancyZones_zonable_tester/FancyZones_zonable_tester.vcxproj

```diff
@@ -80,6 +80,7 @@
       <SDLCheck>true</SDLCheck>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard>stdcpplatest</LanguageStandard>
+      <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">NotUsing</PrecompiledHeader>
```

Comment:
If precompiled header is not used, it should be for all configurations, not only Debug